python/networkx.cpp

```diff
@@ -686,9 +686,17 @@ void ukv::wrap_networkx(py::module& m) {
         py::arg("key"));
     g.def(
         "get_edge_data",
-        [](py_graph_t& g, ukv_key_t v1, ukv_key_t v2) { throw_not_implemented(); },
+        [](py_graph_t& g, ukv_key_t v1, ukv_key_t v2, std::string name) {
+            auto edges = g.ref().edges_between(v1, v2).throw_or_release();
+            auto edge_ids = edges.edge_ids.immutable();
+            auto attrs = read_attributes(g.relations_attrs, edge_ids, name.size() ? name.c_str() : nullptr);
+            if (attrs[0] && attrs[0].size())
+                return py::reinterpret_steal<py::object>(from_json(json_t::parse(attrs[0])));
+            return py::reinterpret_steal<py::object>(py::dict());
+        },
         py::arg("u"),
-        py::arg("v"));
+        py::arg("v"),
+        py::arg("name") = "");
```

Comment:
This argument name doesn't match NetworkX: https://networkx.org/documentation/stable/reference/classes/generated/networkx.MultiDiGraph.get_edge_data.html